python/lsst/meas/astrom/directMatch.py

```diff
@@ -92,22 +92,23 @@ def __init__(self, butler=None, refObjLoader=None, **kwargs):
         self.makeSubtask("sourceSelection")
         self.makeSubtask("referenceSelection")
 
-    def run(self, catalog, filterName=None):
+    def run(self, catalog, filterName=None, epoch=None):
```

Comment:
Pease accept `epoch` as a `DateTime` instead of a float in MJD TAI, in order to avoid an obvious source of errors (incorrect system or scale).